skimage/filters/rank/__init__.py

```diff
@@ -22,6 +22,7 @@
            'subtract_mean',
            'subtract_mean_percentile',
            'median',
+           'median_float',
```

Comment:
It's true that it would be nice to have only one function exposed to the other. Even if the API of the ndimage is different, can you adapt parameters passed to the scikit-image API to determine the corresponding parameters for ndimage, and then test for the dtype of the image within the exposed ``median`` function?